tools/misbehaving-jmx-server/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Use the official Maven image as the base image
+FROM maven:latest
+
+# Set the working directory to /app
+WORKDIR /app
+
+# Copy the pom.xml file and install the dependencies
+COPY pom.xml .
+RUN mvn dependency:resolve
+
+# Copy the source code and build the JAR file
+COPY src/ /app/src/
+RUN mvn clean package assembly:single
+
+RUN ls /app/target
+
+# Use the official OpenJDK image as the base image for the final image
+FROM openjdk:latest
```

Comment:
Should we use build args to test across multiple JVMs? Probably can be saved for a later PR